scripts/Interface/ui/sans_isis/sans_data_processor_window.ui

```diff
@@ -623,7 +632,7 @@ QGroupBox::title {
           <item>
            <widget class="QTabWidget" name="settings_tab_widget">
             <property name="currentIndex">
-             <number>0</number>
+             <number>2</number>
```

Comment:
The settings tab now defaults to being on the adjustment tab when first switched to. This is probably because Qt creator has an annoying feature when it sets whichever tab you are on when you save as the default. Could we switch the default back to General i.e. 0